community/cypher/frontend-3.0/src/main/scala/org/neo4j/cypher/internal/frontend/v3_0/ast/Pattern.scala

```diff
@@ -229,12 +229,31 @@ case class RelationshipChain(element: PatternElement, relationship: Relationship
     rightNode.semanticCheck(ctx)
 }
 
+object InvalidNodePattern {
+  def apply(id: Identifier, labels: Seq[LabelName], properties: Option[Expression])(position: InputPosition) =
+    new InvalidNodePattern(id)(position)
+}
+
+class InvalidNodePattern(val id: Identifier)(position: InputPosition) extends NodePattern(Some(id), Seq.empty, None)(position) {
```

Comment:
Because I only wanted to store the id of the invalid node, not the properties and labels.